### DISCUSSION TITLE
Eigene Berechtigungen entfernen

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#18] [Idea retired / denied]
Wenn man sich selbst aus einem Tenant löscht, kann man sich aktuell einfach entfernen. Vlt. wäre hier eine kurze Meldung sinnvoll, die fragt, ob man sich wirklich selbst aus dem Tenant entfernen möchte.

**Comment by YvesR** [comment]
There is no "tenant-owner", there is a tenant-creator.
There is no real ownership, just the creator inherit all roles on creation.
Afterwards he can invite more users and set roles.

If he removes himself (what often happens, when e.g. a service partner creates a tenant for his customer) then he needs to make sure before another one has necessary roles. If no one left with relevant roles it will be a support issue where we manually do have to set access in ident.services.

**Reply by moritzvolkmar** [reply]
Ok, works for me
